main.go

```diff
@@ -79,3 +114,11 @@ func main() {
 func getWatchNamespace() string {
 	return os.Getenv("WATCH_NAMESPACE")
 }
+
+// loadConfig will populate the configuration
+func loadConfig(f *flag.FlagSet) {
+	_ = f.Parse(os.Args[1:])
```

Comment:
This will swallow invalid flags - maybe panic on errors? This would at least give some hints...
```suggestion
	if err = f.Parse(os.Args[1:]); err != nil; {
		log.Fatalln("parsing flags:", err)
	}
```
Will print the error message and then panic.

Or do the same as below